_pages/members.md

```diff
@@ -304,6 +304,22 @@ order: 1
     </div>
 </div>
 
+<div class="row mt-3">
```

Comment:
I think you can just add yours at line 300, instead of start a new row in the page

Reply:
Hey ftshijt!, good catch! , updated can you check now